2025-07-04 18:55:45,406 ERROR: Exception on /products/sku/1/adjust [POST] [in /home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py:875]
Traceback (most recent call last):
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/app/blueprints/products/sku.py", line 39, in adjust_sku
    quantity = int(request.form.get('quantity'))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
2025-07-04 18:55:56,159 ERROR: Exception on /products/sku/1/adjust [POST] [in /home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py:875]
Traceback (most recent call last):
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/app/blueprints/products/sku.py", line 39, in adjust_sku
    quantity = int(request.form.get('quantity'))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
2025-07-04 18:56:36,647 ERROR: Exception on /products/sku/1/adjust [POST] [in /home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py:875]
Traceback (most recent call last):
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/app/blueprints/products/sku.py", line 39, in adjust_sku
    quantity = int(request.form.get('quantity'))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
2025-07-04 18:56:41,685 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 18:56:54,370 ERROR: Exception on /products/sku/1/adjust [POST] [in /home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py:875]
Traceback (most recent call last):
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/workspace/app/blueprints/products/sku.py", line 39, in adjust_sku
    quantity = int(request.form.get('quantity'))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
2025-07-04 18:57:15,554 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:09:48,651 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:13:14,920 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:13:56,436 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:13:58,363 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:17:35,995 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:21:21,264 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:21:22,937 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:22:20,946 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:22:22,896 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:22:59,785 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:27:01,196 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:27:02,903 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:27:29,795 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:27:31,458 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:27:33,532 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:27:35,850 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:27:37,484 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:28:29,413 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:28:31,247 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:28:33,010 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:29:56,695 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:30:24,739 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:30:27,937 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:30:52,413 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:31:42,850 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:32:28,670 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:46:49,798 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:52:53,802 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 19:52:53,802 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 20:01:22,313 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 20:03:43,810 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 20:03:45,742 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 20:03:53,983 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
2025-07-04 20:04:41,377 INFO: BatchTrack startup [in /home/runner/workspace/app/utils/unit_utils.py:17]
